File: app/Models/Feeder.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Feeder extends Model
{
    protected $fillable = [
        'name',
        'description',
        'gardu_induk_id',
    ];

    public function garduInduk()
    {
        return $this->belongsTo(GarduInduk::class);
    }

    /**
     * Get the status points for this feeder.
     */
    public function statusPoints()
    {
        return $this->hasMany(FeederStatusPoint::class);
    }

    /**
     * Get the keypoints for this feeder.
     */
    public function keypoints()
    {
        return $this->hasMany(FeederKeypoint::class);
    }
}

File: go-backend-pln-monitoring/internal/model/feeder.go
package model

// Feeder represents the feeder model.
type Feeder struct {
	ID             uint   `json:"id" gorm:"primaryKey"`
	Name           string `json:"name"`
	Description    string `json:"description"`
	KeywordAnalogs string `json:"keyword_analogs"`
	GarduIndukID   uint   `json:"gardu_induk_id"`
}
